tests/virtual_machines_test.go

```diff
@@ -62,8 +62,8 @@ var _ = Describe("Virtual Machines", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 		})
-		//2166
 		Context("When the client tries to assign the same MAC address for two different vm. Within Range and out of range", func() {
+			//2166
```

Comment:
In kubevirt, the ID is part of context description, don't we want to follow the same schema? Shoudn't we add some more text to the ID, I get we don't want to reference CNV too much from this project, but 4 digits on their own would make no sense at all to random reader.

Reply:
I just move this as @nikita133 requested.

I will like if he can open a new PR and move from comments to the context itself the test ID.